docs/leaderboard/index.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 1
+sidebar_class_name: sidebar-item-overview
+sidebar_label: Momento Leaderboard
+title: Momento Leaderboard Documentation
+description: Learn about Momento's purpose-built Leaderboard API
+pagination_next: null
+---
+
+## What is Momento Leaderboard?
+
+Momento Leaderboard is a purpose-built leaderboard that supports tens of millions of items and rapid ingestion/querying/updates. While many databases approximate leaderboards via general purpose data structures like sorted set or range queries, Momento Leaderboard is a first-class, full-fledged service that you can quickly and easily integrate with your application. 
+
+## Getting Started with Momento Leaderboard
+
+### Step 1: Create your Momento API key and cache
+
+Go to the [Momento console](https://console.gomomento.com/tokens) and follow the instructions to log in with your email
+address, Google account, or GitHub account.
+
+![Image of Momento console landing page](@site/static/img/getting-started/console.png)
+
+Click on the Create Cache button and create a cache using the cloud provider and region you want to use for your Momento Leaderboard.
+
+![Image of Momento console Create Cache form](@site/static/img/console-create-cache-form.png)
+
+Then create an API key via the [Momento console](https://console.gomomento.com) on the Generate API Key screen by selecting the [API Keys](https://console.gomomento.com/tokens) menu option.
```

Comment:
Instead of typing out the steps, we should link to the auth page that shows how to create an API key.